angu-fixed-header-table.js

```diff
@@ -74,6 +79,7 @@
     function fixedHeader($timeout, $window, $q, $rootScope){
         var attr_prefix = 'fixedHeader';
         var ATTRS = {
+            TABLE_HEIGHT : 'tableHeight', // table height
```

Comment:
You are not using the prefix since we already had it that way, right? i.e to maintain compatibility?